.github/CODEOWNERS

```diff
@@ -86,27 +85,26 @@
 # PRLabel: %Communication - Rooms
 /sdk/communication/azure-communication-rooms/       @minnieliu @paolamvhz @alexokun @Mrayyan @shwali-msft @allchiang-msft @mikehang-msft
 
-# AzureSDKOwners: @lmolkova
+# AzureSdkOwners: @lmolkova
 # ServiceLabel: %Container Registry
 # PRLabel: %Container Registry
 /sdk/containerregistry/                              @lmolkova @alzimmermsft @Azure/azsdk-acr
 
-# AzureSDKOwners: @alzimmermsft
+# AzureSdkOwners: @alzimmermsft
 # ServiceLabel: %Azure.Core
 # PRLabel: %Azure.Core
 /sdk/core/                                           @alzimmermsft @srnagar @anuchandy @lmolkova @vcolin7 @mssfang @jonathangiles @billwert
 
-# AzureSDKOwners: @anuchandy @conniey @lmolkova
+# AzureSdkOwners: @anuchandy @conniey @lmolkova
 # ServiceLabel: %Azure.Core.AMQP
 # PRLabel: %Azure.Core.AMQP
 /sdk/core/azure-core-amqp/                           @anuchandy @conniey @lmolkova
 
-# AzureSDKOwners: @lmolkova
-# ServiceLabel: %Azure.Core.Tracing.OpenTelemetry
-# PRLabel: %Azure.Core.Tracing.OpenTelemetry
+# AzureSdkOwners: @lmolkova
+# ServiceLabel: %OpenTelemetry
```

Comment:
Seems we should not remove `Azure.Core.Tracing.`

Reply:
The codeowners linter didn't like it since it's not consistent with the other repos.  I merged it to OpenTelemetry to make the linter happy since that's the label the other repos use

Reply:
It can either be %Azure.Core or %OpenTelemetry right now.  The linter doesn't like %Azure.Core.Tracing since it's not in the main-cross-repo list at the moment.